phylum_project/src/lib.rs

```diff
@@ -51,7 +51,8 @@ impl ProjectConfig {
                 .iter()
                 .map(|depfile| {
                     let path = self.root.join(&depfile.path);
-                    DepfileConfig::new(path, depfile.depfile_type.clone())
+                    let dunce_path = dunce::simplified(&path);
+                    DepfileConfig::new(dunce_path, depfile.depfile_type.clone())
```

Comment:
The variable name `dunce_path` doesn't really mean anything (unless you're calling the path stupid...). You can just use variable shadowing instead:

```suggestion
                    let path = dunce::simplified(&path);
                    DepfileConfig::new(path, depfile.depfile_type.clone())
```

Or avoid the variable altogether (reformatting is probably needed for this one):

```suggestion
                    DepfileConfig::new(dunce::simplified(path), depfile.depfile_type.clone())
```

Reply:
Thank you for the suggestions. I learned something with the first one but went with the second one.